packages/stores/types.ts

```diff
@@ -232,7 +235,7 @@ type TTradingPlatformAvailableAccount = {
 type TAvailableCFDAccounts = {
     availability: 'Non-EU' | 'EU' | 'All';
     description: string;
-    icon: 'Derived' | 'Financial' | 'DerivX' | 'SwapFree' | 'Ctrader';
+    icon: 'Derived' | 'Financial' | 'DerivX' | 'SwapFree' | 'CTrader';
```

Comment:
Will it effect the icons changes?

Reply:
As I check it once, it did not. Will check it again.